plugins/editor/class.editor.plugin.php

```diff
@@ -1508,11 +1508,9 @@ public function utilityController_mediaThumbnail_create($sender, $media_id) {
                 'ThumbPath' => $filepath_parsed['SaveName']
             ));
 
-            // Remove cf scratch copy, typically in cftemp, if there was actually a file pulled in from CF.
-            if (strpos($local_path, 'temp') !== false) {
-                if (!unlink($local_path)) {
-                    // Maybe add logging for local cf copies not deleted.
-                }
+            // Remove cf scratch copy, typically in /uploads/cftemp/ or /uploads/cloudtemp/, if there was actually a file pulled in from CF.
```

Comment:
We should probably use the constant `PATH_UPLOADS` instead of hardcoding `/uploads ` in the regex since our Cloud storage plugins uses that constant as well.
This would prevent issues for users setting `PATH_UPLOADS` to somewhere else.